Support jacoco aggregate

7sean68 opened the pull request:
closes #401

ryanluker commented on package.json:
```diff
@@ -289,10 +289,10 @@
     "typescript": "4.5.5"
   },
   "dependencies": {
+    "@7sean68/jacoco-parse": "^2.2.0",
```
Probably just ping this to `2.2.0` instead.
```suggestion
    "@7sean68/jacoco-parse": "2.2.0",
```

ryanluker commented on test/extension.test.ts:
```diff
@@ -219,6 +219,45 @@ suite("Extension Tests", function() {
         decorationSpy.restore();
     });
 
+    test("Run display coverage on java files from jacoco-aggregate report @integration", async () => {
```
Excellent, thanks a ton for adding tests 👍🏻.

ryanluker commented:
@7sean68 We might want to add "Multi-Module Java" to the example projects list in the README.
https://github.com/ryanluker/vscode-coverage-gutters#officially-supported-languages-backed-by-tests-ran-nightly

7sean68 commented:
can you please not merge it now?

7sean68 commented:
something I need to check first

ryanluker commented:
> something I need to check first

Sure thing, I have just need hitting the "approve and run" button to get the CI workflow to run for yeah.

7sean68 commented:
I added the example to readme.
I also added a notice to the copied test files from an Apache repo can you check if everything is ok?
It's ready now you can merge if everything is ok. Thanks for the wait 😄

ryanluker commented:
> I added the example to readme. I also added a notice to the copied test files from an Apache repo can you check if everything is ok? It's ready now you can merge if everything is ok. Thanks for the wait 😄

Excellent, I will try to take a final look within the next week.

ryanluker reviewed (approved):
Looking great thanks again for the contribution 🦾 .
Re Apache license, the examples folder is not distributed with the extension into the vscode marketplace so the MIT license will still be good there, thanks for checking though!

ryanluker commented:
> I think Apache license is compatible with MIT license anyway. So, even if the example are shipped the notice is enough and it doesn't affect the whole repo being MIT ... BTW that's why I paused merging the first time. I had to contact the author of the repo as I discovered it wasn't licensed ... thankfully he added the license and I updated the files to add the notice to not cause any problems for the repo.
> 
> Thanks for the great work creating and maintaining this helpful project. I use it as a main extension in vscode for java development.

Excellent, thanks for going that extra mile to get a license added!
Great to hear that the extension is providing value for your development work, mission accomplished!

Thanks again for the contribution, this helps keep the project running smoothly.
